docs/integrators/maiar-integration-react-native-guide.md

```diff
@@ -0,0 +1,344 @@
+# Maiar integration documentation
+
+This is the official documentation presented by the [Sense4FIT team](https://sense4fit.io/) for **Elrond developers** who want to integrate with the Maiar application.
+
+# How to connect to Maiar and sign transactions using React-native
+
+Currently all you need in order to make this work are the next libraries:
+
+- [@walletconnect/client](https://www.npmjs.com/package/@walletconnect/client)
+- [@elrondnetwork/erdjs](https://www.npmjs.com/package/@elrondnetwork/erdjs)
+- [@elrondnetwork/erdjs-network-providers](https://www.npmjs.com/package/@elrondnetwork/erdjs-network-providers)
+
+## WalletConnectProvider
+
+You can use most of the `WalletConnectProvider` located [here](https://github.com/ElrondNetwork/elrond-sdk-erdjs-wallet-connect-provider/blob/main/src/walletConnectProvider.ts) already implemented by the Elrond team, the only thing that we had to provide to the constructor was the `clientMeta` prop.
+
+```ts
+import WalletClient from "@walletconnect/client";
```

Comment:
can you please make use of dapp-core instead of using walletconnect directly? apply the refactoring to the entire guide

Reply:
In react-native you can't use dapp-core, that's the reason why we had to use the walletConnect directly. Dapp core has browser dependencies which are incompatible with react-native.

Reply:
what about https://github.com/ElrondNetwork/elrond-sdk-erdjs-wallet-connect-provider/ ?